packages/purpose-process/src/services/purposeService.ts

```diff
@@ -220,14 +222,6 @@ async function retrieveTenantKind(
   return tenant.kind;
 }
 
-async function retrieveActiveDelegation(
```

Comment:
Deleted since we usually use retrieve when we want to throw 404 if not found, since this only calls the readmodel might be confusing